### ISSUE TITLE
Wrong character length for Latin-1

### ISSUE THREAD

**Issue by manuelsucco** [tadashi-aikawa/markowl#4]
It seems to me that when using characters from the [Latin-1-Supplement](https://en.wikipedia.org/wiki/Latin-1_Supplement_(Unicode_block)) (I was using German Umlaute ä, ü, ö and the superscript ²) the width is falsely returned as 2 (should be 1). Therefore the formatted table cell is too short.

This is the table after `Format Table`:
```markdown
| x   | y   |
| --- | --- |
| ä  | y   |
| ö  | y   |
| ü  | y   |
| ²  | y   |
```

This seems weird, since you already include character codes 0x0 through 0x81 in the [`width`-function](https://github.com/tadashi-aikawa/markowl/blob/master/src/main/kotlin/net/mamansoft/markowl/util/StringUtil.kt).

---
I'm using PHPStorm 2021.2 with Markowl v0.7.1.

**Comment by tadashi-aikawa** [comment]
Hi, @manuelsucco. Thank you for the kind report!

I will fix the width function to return 1 when a code point matchs `0xa1..0xff`.

**Comment by manuelsucco** [comment]
Awesome! Thanks a lot.

Apart from that bug, I really love your extension! Great work.

**Comment by tadashi-aikawa** [comment]
Hi, @manuelsucco.
I have released a new version v0.7.2.

![markowl](https://user-images.githubusercontent.com/9500018/130073050-ec1ba7f9-1365-476c-8622-0cc44ef1e6d8.gif)

It would probably be tomorrow or day after tomorrow that you can upgrade it because JetBrains needs to approve it.

Best regards.

**Comment by manuelsucco** [comment]
Wow. That was quick. Thank you so much @tadashi-aikawa.

I'll test it tomorrow. 😊
